setup.py

```diff
@@ -1,6 +1,21 @@
+import setuptools
 from setuptools import setup
 from versioningit import get_cmdclasses
 
+if int(setuptools.__version__.split(".")[0]) < 61:
```

Comment:
are we sure that `__version__` attribute for setuptools and pip exist on all the versions that realistically can be used? i understand that this is soooooo unlikely :)

Reply:
For setuptools this has been there at least since 34.0.0 which was the earliest version that I can actually install.

For pip at least since 10.0 which was the earliest version I could get to work

Reply:
great!